res/openvpn-gui-res-ua.rc

```diff
@@ -0,0 +1,341 @@
+    LTEXT "Підтвердження паролю:", 173, 6, 42, 85, 10
+    EDITTEXT ID_EDT_PASS_CUR, 95, 6, 90, 12, ES_PASSWORD | ES_AUTOHSCROLL
+    EDITTEXT ID_EDT_PASS_NEW, 95, 23, 90, 12, ES_PASSWORD | ES_AUTOHSCROLL
+    EDITTEXT ID_EDT_PASS_NEW2, 95, 39, 90, 12, ES_PASSWORD | ES_AUTOHSCROLL
+    PUSHBUTTON "OK", IDOK, 40, 59, 50, 14
+    PUSHBUTTON "Відміна", IDCANCEL, 103, 59, 50, 14
+    LTEXT "", ID_TXT_KEYFORMAT, 0, 0, 0, 0
+    LTEXT "", ID_TXT_KEYFILE, 0, 0, 0, 0
+END
+
+/* Proxy Settings Dialog */
+ID_DLG_PROXY DIALOG 6, 18, 249, 104
+STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER
+CAPTION "Проксі-сервер"
+FONT 8, "Microsoft Sans Serif"
+LANGUAGE LANG_UKRAINIAN, SUBLANG_DEFAULT
+BEGIN
+    GROUPBOX "   ", 201, 6, 46, 235, 52
+    AUTORADIOBUTTON "Використати налаштування з файла конфігурації OpenVPN", ID_RB_PROXY_OPENVPN, \
+                    13, 16, 210, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Використати системні налаштування проксі-сервера", \
+                    ID_RB_PROXY_MSIE, 13, 31, 200, 10
+    AUTORADIOBUTTON "Налаштування проксі вручну", ID_RB_PROXY_MANUAL, 13, 46, 79, 10
+    AUTORADIOBUTTON "HTTP-проксі", ID_RB_PROXY_HTTP, 20, 62, 90, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "SOCKS-проксі", ID_RB_PROXY_SOCKS, 120, 62, 90, 10
+    LTEXT "Адреса:", ID_TXT_PROXY_ADDRESS, 20, 77, 25, 10
+    RTEXT "Порт:", ID_TXT_PROXY_PORT, 171, 77, 20, 10
+    EDITTEXT ID_EDT_PROXY_ADDRESS, 47, 75, 118, 12, ES_AUTOHSCROLL
+    EDITTEXT ID_EDT_PROXY_PORT, 196, 75, 30, 12, ES_AUTOHSCROLL
+END
+
+/* General Settings Dialog */
+ID_DLG_GENERAL DIALOGEX 6, 18, 249, 104
+STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER
+CAPTION "Загальні налаштування"
+FONT 8, "Microsoft Sans Serif"
+LANGUAGE LANG_UKRAINIAN, SUBLANG_DEFAULT
+BEGIN
+    GROUPBOX "Інтерфейс користувача", 201, 6, 12, 235, 30
+    LTEXT "Мова:", ID_TXT_LANGUAGE, 17, 25, 23, 12
+    COMBOBOX ID_CMB_LANGUAGE, 42, 23, 186, 400, CBS_DROPDOWNLIST | WS_TABSTOP
+END
+
+/* About Dialog */
+ID_DLG_ABOUT DIALOG 6, 18, 249, 104
+STYLE WS_POPUP | WS_CAPTION | WS_SYSMENU | DS_CENTER
+CAPTION "Про программне забеспечення OPENVPN"
+FONT 8, "Microsoft Sans Serif"
+LANGUAGE LANG_UKRAINIAN, SUBLANG_DEFAULT
+BEGIN
+    ICON ID_ICO_APP, 0, 8, 16, 21, 20
+    LTEXT "OpenVPN GUI v" PACKAGE_VERSION " - графічний інтерфейс Windows для OpenVPN\n" \
+          "Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n" \
+          "http://openvpn.se/", 0, 36, 15, 206, 26
+    LTEXT "OpenVPN - программне забеспечення, для безпечного тунелювання IP-мереж " \
+          "через єдиний UDP-порт із підтримкой підтверження подлінності сесій " \
+	  "і пересилкой(обміном) ключів SSL/TLS, шифруванням, аутентіфікацієй " \
+	  "і з компрессією пакетів.\n" \
+          "\n" \
+          "Copyright (C) 2002-2005 OpenVPN Solutions LLC <info@openvpn.net>\n" \
+          "Supported by - За підтримки - При подддержке http://picku.pp.ua/\n" \
```

Comment:
I suggest "Translated by" or "Translation maintained by" instead of "Supported by". Now this gives the impression that OpenVPN-GUI is officially supported by picku.pp.ua (if the language is Ukrainian).